packages/wdio-allure-reporter/src/reporter.ts

```diff
@@ -31,6 +31,7 @@ import type {
 import {
     TYPE as DescriptionType
 } from './types.js'
+import type { BeforeCommand, CustomCommand } from 'node_modules/@wdio/reporter/build/types.js'
```

Comment:
We should export these types if we use them elsewhere
```suggestion
import type { BeforeCommand, CustomCommand } from '@wdio/reporter'
```

Reply:
Done